### PR TITLE
add ajax for build page;

### PR DESCRIPTION
**Pull request by matthewalbani**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -47,6 +49,19 @@ jQuery(document).ready(function() {
   jQuery("abbr.timeago").timeago();
 });
 
+Kochiku.updateBuildParts = function(renderTime) {
+  $.getJSON(document.URL + '/updated_parts', { modified_time: renderTime }, function( data ) {
+    console.log(data);
```
Comment (robolson) [inline app/assets/javascripts/application.js]: debugging line

File: app/controllers/builds_controller.rb [hunk 2]
```diff
@@ -131,6 +131,27 @@ def modified_time
     end
   end
 
+  def updated_parts
```
Comment (robolson) [inline app/controllers/builds_controller.rb]: I am not liking the name `updated_parts`. Perhaps `refresh_build_part_info` ?

File: app/views/layouts/application.html.haml [hunk 3]
```diff
@@ -15,6 +15,8 @@
       = favicon
     - else
       = favicon_link_tag '/favicon.ico'
+    = yield :head_meta
+    = yield :head_js
```
Comment (robolson) [inline app/views/layouts/application.html.haml]: what are these lines for? I can't tell if they are being used.

Comment (matthewalbani) [reply app/views/layouts/application.html.haml]: they survived from action_cable ill remove

File: app/controllers/builds_controller.rb (hunk 2 above)
Comment (matthewalbani) [reply app/controllers/builds_controller.rb]: ill change

File: app/assets/javascripts/application.js [hunk 4]
```diff
@@ -33,12 +33,14 @@ Kochiku.delayedRefresh = function(updateInfo) {
...
     if($('input#refresh').is(':checked')) {
       $.getJSON(document.URL + '/modified_time', function( data ) {
-        var buildTime = new Date(Date.parse(data));
-        if(buildTime > updateInfo.renderTime) {
-          window.location.reload();
+        var buildTime = Date.parse(data);
+        var renderTime = updateInfo.renderTime
+        if(buildTime > renderTime) {
+          Kochiku.buildInfo.renderTime = buildTime
+          Kochiku.updateBuildParts(renderTime)
```
Comment (robolson) [inline app/assets/javascripts/application.js]: I believe semicolons are optional in JS now but leaving them off does not match the style of the rest of the file.

**Pull request merged**